index.html

```diff
@@ -37,6 +37,177 @@ <h1 class="intro">I’m a software developer</h1>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul> 
+                    <div class="project">
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="work-item">
+                <div class="snapshot">
+                </div>
+                <div class="details">
+                    <h2>Multi-Post Stories Gain+Glory</h2>
+                    <ul class="tech-stack">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul> 
+                    <div class="project">
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="work-item">
+                <div class="snapshot">
+                </div>
+                <div class="details">
+                    <h2>Multi-Post Stories Gain+Glory</h2>
+                    <ul class="tech-stack">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <div class="project">
+                        <button>See Project</button>
+                    </div> 
+                </div>
+            </div>
+
+            <div class="work-item">
+                <div class="snapshot">
+                </div>
+                <div class="details">
+                    <h2>Multi-Post Stories Gain+Glory</h2>
+                    <ul class="tech-stack">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul> 
+                    <div class="project">
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="work-item">
+                <div class="snapshot">
+                </div>
+                <div class="details">
+                    <h2>Multi-Post Stories Gain+Glory</h2>
+                    <ul class="tech-stack">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul> 
+                    <div class="project">
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+      
+    </section>
+    <section class="about_section">
+        <div class="about-details">
+            <h1>About me</h1>
```

Comment:
- [ ] The page is supposed to only have one heading element `h1` for semantic reasons, Kindly you may consider using `h2` ...`hx`